src/App.css

```diff
@@ -35,18 +36,123 @@ nav ul li a {
   font-weight: bold;
 }
 
+.book-list {
+  display: flex;
+  flex-direction: column;
+  padding: 0 5%;
+}
+
 .book-card {
+  background-color: #fff;
+  margin-top: 2%;
+  list-style: none;
+  padding: 2%;
+}
+
+.card-container {
   display: grid;
-  gap: 25%;
-  margin-top: 3%;
+  grid-template-columns: 2fr 1fr 1px 1fr;
+}
+
+.btn-ul {
+  list-style: none;
+  display: flex;
+  flex-direction: row;
```

Comment:
- [ ] Kindly use resuable classes for rpeating CSS rules. For example `display: flex;` has been reused in this file. consider making a reusable class and attaching it to the affected elements